### PR TITLE
Aligning with new navigation

### PR DESCRIPTION
**Pull request by mpanhuijsen**
See https://confluence.sdl.com/display/SDLID/Updating+a+DevDocs+section

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,8 +1,20 @@
 # Welcome to the Tridion Sites developer documentation
 
-## [Learn more about Tridion Sites APIs](apiconcepts/overview.md) 
+Here you’ll find all the information you need to get started developing against your own instance of Tridion Sites. Discover quick guides explaining how to build, develop and deploy your app, plus hints and tips to help you make the most of our APIs.
+
+Should you have any ideas as to how we can improve this documentation, any new topics you think we should include or hints and tips you would like to share with the developer community, please let us know.
```
Comment (jglandman) [inline index.md]: Using "Should you have..." is more formal than simply saying "If you have...", so it's ok but if begs the question on what tone we wish to use.  I'd also simplify this sentence a bit...
_Let us know if you have ideas for improving this documentation, new topics you think we should include or tips you would like to share with the developer community._

File: index.md [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+
+To let us know what your suggested improvements are you can do this by using the "**Improve this Doc**" link on the top right of every page.  You will need your own github account for this because this action will create a fork of the repository in your account where you can make the changes and then submit a pull request for us to review and accept.
```
Comment (jglandman) [inline index.md]: If you do decide to take my other suggestion, i'd also remove the first part of this sentence.

**Review by jglandman — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (mpanhuijsen) [reply index.md]: Reworded, would you mind checking again?

File: index.md (hunk 2 above)
Comment (mpanhuijsen) [reply index.md]: Reworded, would you mind checking again?

**Review by jglandman — APPROVED** [review]
Looks good.

**Pull request merged**